[ADD] Inventory: add stock valuation cheat sheet (#14906)
odoo/documentation: content/applications/inventory_and_mrp/inventory/inventory_valuation/cheat_sheet.rst

:code-column:
:custom-css: valuation.css
:custom-js: misc.js,valuation-data.js,valuation-journal.js,valuation-accounting.js

=====================
Valuation cheat sheet
=====================

.. rst-class:: full-width

   .. important::
      This documentation is for Odoo 19 or later.

>       :ref:`Discover why we changed. <changes-in-19>`

Reviewer: ```suggestion
```

   process happens usually once a year for SMEs, or once a month for larger companies.

.. role:: good
.. role:: meh
.. role:: bad

.. h:div:: feature-table doc-aside

  +------------------+------------+-----------+
  |                  | Accounting | Inventory |
  +==================+============+===========+
  | Purchase Order   | :meh:`/`   | :meh:`/`  |
  +------------------+------------+-----------+
  | Receipt          | :meh:`/`   | :good:`✓` |
  +------------------+------------+-----------+
  | Vendor Bill      | :good:`✓`  | :meh:`/`  |
  +------------------+------------+-----------+
  | Sales Order      | :meh:`/`   | :meh:`/`  |
  +------------------+------------+-----------+
  | Customer Invoice | :good:`✓`  | :meh:`/`  |
  +------------------+------------+-----------+
  | Delivery         | :meh:`/`   | :good:`✓` |
  +------------------+------------+-----------+
  | Closing Entry    | :good:`✓`  | :meh:`/`  |
  +------------------+------------+-----------+

Accounting Methods
==================

There are two accounting practices on how to maintain your accounts:

**Periodic:** Post vendor bills as expenses by nature, and update stock valuation in the closing

> entry by reducing expenses (stock variation). It’s the best practice in Europe.

Reviewer: ```suggestion
entry by reducing expenses (stock variation). This is the best practice in Europe.
```

**Perpetual:** Post vendor bills as assets (stock valuation), report expenses when goods are sold

> (cost of goods sold). It’s the best practice in countries that follow Anglo-Saxon accounting, like

Reviewer: ```suggestion
(cost of goods sold). This is the best practice in countries that follow Anglo-Saxon accounting, like
```

calculated.

.. h:div:: doc-aside

  .. image:: cheat_sheet/unit-cost.png

Total value
~~~~~~~~~~~

By opening :guilabel:`Total Value`, you can see all incoming quantities for which you still have a
remaining quantity and the value used for their valuation. In AVCO or standard cost, the used value
is always the current average unit cost. In FIFO, remaining units from each previous incoming move
keep their own individual valuation.

In FIFO or AVCO remaining quantities from a previous incoming move can have their value adjusted if
necessary.

.. image:: cheat_sheet/total-value.png
   :scale: 90%

.. h:div:: doc-aside

  .. image:: cheat_sheet/total-value2.png

In Accounting
-------------

Open :menuselection:`Review --> Inventory Valuation` to have a look at the difference between the
accounting stock value and the current inventory value recorded thanks to the incoming moves with a
remaining quantity.

Click on :guilabel:`Generate Entry` to get a new accounting entry to review and post.

Open
:menuselection:`Review --> Invoices not received, Invoices to be issued, Prepaid expenses and Deferred Revenues`

> to easily record these entries.

Reviewer 2: It will arrive soon ...

:code-column:
:custom-css: valuation.css
:custom-js: misc.js,valuation-data.js,valuation-journal.js,valuation-accounting.js

=====================
Valuation cheat sheet
=====================

.. rst-class:: full-width

   .. important::
      This documentation is for Odoo 19 or later.
      :ref:`Discover why we changed. <changes-in-19>`

> 

Reviewer 3: oh okay! definitely ignore my suggestion since it won't render properly (:

   .. placeholder

.. _journal-entries:

Journal Entries Configuration
=============================

.. h:div:: journal-entries doc-aside

    .. placeholder

Reporting
=========

In Inventory
------------

Open :menuselection:`Reporting --> Stock` to view your current inventory level and valuation for
each product, or to review historical data as of a previous date.

.. h:div:: doc-aside

  .. image:: cheat_sheet/valuation-stock.png

Unit cost
~~~~~~~~~

You can click on :guilabel:`Unit Cost` to check all existing updates and their origins. In
:abbr:`AVCO (Average Cost)` this allows you to understand how the currently used value was
calculated.

.. h:div:: doc-aside

  .. image:: cheat_sheet/unit-cost.png

Total value
~~~~~~~~~~~

By opening :guilabel:`Total Value`, you can see all incoming quantities for which you still have a
remaining quantity and the value used for their valuation. In AVCO or standard cost, the used value
is always the current average unit cost. In FIFO, remaining units from each previous incoming move
keep their own individual valuation.

In FIFO or AVCO remaining quantities from a previous incoming move can have their value adjusted if

> necessary.

Reviewer 3: ```suggestion
The :guilabel:`Total Value` represents the cost of remaining stock, along with the value used for their valuation
- In AVCO or standard cost, the total value is always the current average unit cost
- In FIFO, remaining units from each incoming move retain their original valuation

In FIFO or AVCO, the value of remaining quantities from earlier moves can have their value adjusted if
necessary.
```
